app/LinkLiteAgent/linkliteagent/main.py

```diff
@@ -1,33 +1,66 @@
 import sys
-import os
 import logging
-import time
-import asyncio
+import pika
+import linkliteagent.message_queue as mq
+from linkliteagent.db_manager import SyncDBManager
+from linkliteagent.db_logging import SyncLogDBHandler
+from linkliteagent.query import query_callback
 
 
-async def loop():
-    """The main method"""
-    running = True
-    # send the logging to stdout
-    handler = logging.StreamHandler(sys.stdout)
-    # set up the logger
-    logger = logging.getLogger("linkliteagent")
-    logger.setLevel(logging.INFO)
-    logger.addHandler(handler)
-
-    while running:
-        try:
-            # hello world
-            logger.info(f"Hello World @ {time.ctime()}")
-            time.sleep(5)
-        except KeyboardInterrupt:
-            # shut down on Ctrl+C
-            logger.warning(f"{os.linesep}Shut down @ {time.ctime()}")
-            running = False
+async def async_main():
+    """An asynchronous version of the main function"""
+    pass
 
 
 def main():
-    asyncio.run(loop())
+    QUEUE_NAME = "link-like-queue"
```

Comment:
I know we only have one right now, but all queues in our rabbitmq instance are:
- part of link lite
- queues
so the name is not meaningful.

Would be better to name it for its purpose or what it contains:
```suggestion
    QUEUE_NAME = "jobs"
```